Documentation/operations/upgrade.rst

```diff
@@ -386,6 +386,10 @@ Changed Metrics
   met (e.g: NodePort service implementation in BPF is disabled or socket load-balancing
   is disabled), it will fail to initialize and will log an error instead of silently
   fall back to iptables based masquerading.
+* The Go types for Network policies have changed to accept both integers and strings in
+  the ICMP protocol type. Specifically, the ICMP ``type`` field can be either
```

Comment:
```suggestion
* The ICMP ``type`` field in Network Policy now can be either
```
Or something similar - I think the upgrade notes should highlight the change in the CRD (user-facing) rather than in Go types (implementation detail).

Reply:
Thanks. I addressed to both of your comment at https://github.com/cilium/cilium/pull/30330/commits/5ca340c7ac1b68daf83e90f3f25fa8e18266d1a2.